### PR TITLE
Fix book 2 image numbering, book 3 captions

### PR DESCRIPTION
**Pull request by hollasch**
Quick render/image numbering & caption pass.

### CODE REVIEW

**Comment by trevordblack** [comment]
```
I just grabbed a random earth map from the web -- any standard projection will do for our purposes.

  ![Image 14: earthmap.jpg](../images/earthmap.jpg class=pixel)
```
  
```
  If the photo comes back with a large cyan sphere in the middle, then `stb_image` failed to find your
Earth map photo. The program will look for the file in the same directory as the executable. Make
sure to copy the Earth into your build directory, or rewrite `earth()` to point somewhere else.

  ![Image 15: Earth-mapped sphere](../images/img-2.15-earth-sphere.png class=pixel)
```

etc.

**Comment by trevordblack** [comment]
Since you opened this can of worms, rename all of the comments for images, figures, and listings
or create a new issue specifically for that, and I'll do it

**Comment by hollasch** [comment]
Gah! Forgot to add the book changes to the commit.

File: books/RayTracingTheRestOfYourLife.html [hunk 1]
```diff
@@ -2531,7 +2531,7 @@
 This yields an exactly matching result so all we’ve done so far is move some computation up into the
 `cosine_pdf` class:
 
-  ![Image 9: Cornell box with a cosine density _pdf_
+  ![Image 9: Cornell box with a cosine density PDF
   ](../images/img-3.06-cornell-cos-pdf.jpg class=pixel)
```
Comment (trevordblack) [inline books/RayTracingTheRestOfYourLife.html:2535]: Can you rename the images in book 3 to be sequential (i.e. match the `Image #` with the `img-3.##`) ?

**Review by trevordblack — APPROVED** [review]
Approve with changes to book 3 image names.

File: books/RayTracingTheRestOfYourLife.html (hunk 1 above)
Comment (hollasch) [reply books/RayTracingTheRestOfYourLife.html:2535]: Hmmm, image 4 and image 5 both use `img-3.02-cornell-refactor2.jpg`. Note sure how to untangle.

**Comment by hollasch** [comment]
Note that renders 4 &5 both use `img-3.04-cornell-refactor2.jpg`.

**Pull request merged**